mavros/src/plugins/command.cpp

```diff
@@ -25,6 +25,7 @@
 #include <mavros_msgs/CommandTOL.h>
 #include <mavros_msgs/CommandTriggerControl.h>
 #include <mavros_msgs/CommandTriggerInterval.h>
+#include <mavros_msgs/VtolTransition.h>
```

Comment:
The service should be called `CommandTransition` so to match the other commands.

Reply:
@Jaeyoung-Lim please also change the service name.

Reply:
Done!